### ISSUE TITLE
Testing this driver with Medion S2218 / Z3735F netbook.

### ISSUE THREAD

**Issue by hzulla** [anthonywong/rtl8723bs#1]
Hi,

I'd love to help with testing this on my Medion S2218 with its Z3735F. Having this driver in mainline would be wonderful.

Currently running Yakkety Yak. Compiled using [the instructions from here](https://bugzilla.kernel.org/show_bug.cgi?id=113161).

Kernel 4.4.0-9136: Compiled, worked, crashed after a few minutes. How to give you a meaningful crash report?

**Comment by hzulla** [comment]
Tried compiling for 4.8 with ppa:canonical-kernel-team/unstable.

```
make ARCH=x86_64 CROSS_COMPILE= -C /lib/modules/4.8.0-6-generic/build M=/root/src/rtl8723bs  modules
make[1]: Entering directory '/usr/src/linux-headers-4.8.0-6-generic'
  CC [M]  /root/src/rtl8723bs/os_dep/linux/ioctl_linux.o
/root/src/rtl8723bs/os_dep/linux/ioctl_linux.c: In function 'rtw_ioctl_wext_private':
/root/src/rtl8723bs/os_dep/linux/ioctl_linux.c:13858:5: error: implicit declaration of function 'is_compat_task' [-Werror=implicit-function-declaration]
  if(is_compat_task())
     ^~~~~~~~~~~~~~
cc1: some warnings being treated as errors
scripts/Makefile.build:289: recipe for target '/root/src/rtl8723bs/os_dep/linux/ioctl_linux.o' failed
make[2]: *** [/root/src/rtl8723bs/os_dep/linux/ioctl_linux.o] Error 1
Makefile:1489: recipe for target '_module_/root/src/rtl8723bs' failed
make[1]: *** [_module_/root/src/rtl8723bs] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-4.8.0-6-generic'
Makefile:1680: recipe for target 'modules' failed
make: *** [modules] Error 2
```

**Comment by anthonywong** [comment]
Hi, for your first problem on 4.4, you can post your /var/log/kern.log. As you are running Ubuntu, you can also try the dkms package I posted in https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1581711.

**Comment by hzulla** [comment]
Thanks for your response!

I watched `/var/log/kern.log` this time, but it's inconclusive.

[kern.log.txt](https://github.com/anthonywong/rtl8723bs/files/464035/kern.log.txt)

This is the part of kern.log after compiling the driver and doing `modprobe`, then using the system for a few moments until it crashed. (Some lines of the log didn't end up on disk, I took a screenshot of the actual content while watching the `tail -f` and there were are few lines more that didn't get saved, but those are just a repeat of these here.)

The system crashed for no obvious reason while I was browsing the net with Firefox. The screen was still on, but the content frozen, the computer remained unresponsive (no mouse, no keyboard) and I had to toggle power to restart it. I checked `/var/log/syslog` and found no helpful entry there, either.
